aws/data_source_aws_dynamodb_table.go

```diff
@@ -145,31 +158,38 @@ func dataSourceAwsDynamoDbTable() *schema.Resource {
 					return strings.ToUpper(value)
 				},
 			},
-			"tags": tagsSchemaComputed(),
-			"ttl": {
-				Type:     schema.TypeSet,
+			"stream_arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"stream_label": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"server_side_encryption": {
+				Type:     schema.TypeList,
 				Computed: true,
 				MaxItems: 1,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
-						"attribute_name": {
+						"enabled": {
+							Type:     schema.TypeBool,
+							Computed: true,
+						},
+						"sse_type": {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
-						"enabled": {
-							Type:     schema.TypeBool,
+						"kms_master_key_id": {
+							Type:     schema.TypeString,
 							Computed: true,
 						},
 					},
 				},
 			},
-			"write_capacity": {
-				Type:     schema.TypeInt,
-				Computed: true,
-			},
-			"server_side_encryption": {
+			"tags": tagsSchemaComputed(),
+			"point_in_time_recovery": {
```

Comment:
Added `point_in_time_recovery` as the documentation says that the data source returns all the same attributes as the resource.